pom.xml

```diff
@@ -32,6 +32,7 @@
     <impsort.goal>sort</impsort.goal>
     <jacoco.agent.argLine/>
     <spotbugs.failOnViolation>true</spotbugs.failOnViolation>
+    <version.org.mockito>3.1.0</version.org.mockito>
```

Comment:
In a different PR, later on, we'll want to moved into kie-parent

Reply:
Yes but upgrading the whole KIE to Mockito 3.x is a completely different story :slightly_smiling_face: